13/umbraco-deploy/deployment-workflow/import-with-migrations.md

```diff
@@ -87,6 +87,34 @@ internal class ArtifactMigratorsComposer : IComposer
 }
  ```
 
+### Import and migration flow
+
+When an import is started, the following happens:
+1. Artifact signatures are read from the import provider (using `IArtifactImportProvider.GetArtifactSignatures()`).
```

Comment:
Before I start working on this list, I need to know: Is it steps that the user needs to go through, or is it an explanation of what happens during a migration/import? 🤔

Reply:
This is an explanation of the things that happen and should give users a better understanding of the (order of) operations.

We also hadn't documented the notifications yet or the fact a default `ValidateArtifactImportDependenciesNotificationHandler` is added by Deploy. We might want to add a separate section about notifications though, as we also have notifications that happen during export (`ArtifactExportingNotification` and `ArtifactExportedNotification`).

Reply:
Thanks. I wont touch the list then 💪 

It would be great to have a new section about notifications.
Let me know what you need from us to set this up!